acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -57,6 +57,8 @@ def report_query(node)
 end
 
 with_puppet_running_on(master, {}) do
+  masterfqdn = on(master, '/opt/puppetlabs/bin/facter fqdn').stdout.chomp
```

Comment:
It wasn't necessary in those curl calls, and I can't just define a top-level variable since there's no outer class/module.  Since it was only necessary in these calls, and since I could define the variable in this scope, I just did that and moved on.